clients/admin-ui/src/constants.ts

```diff
@@ -119,6 +119,9 @@ export const LOGIN_ROUTE = "/login";
 export const USER_MANAGEMENT_ROUTE = "/user-management";
 export const CONNECTION_ROUTE = "/connection";
 export const CONNECTION_TYPE_ROUTE = "/connection_type";
+export const DATAMAP_ROUTE = "/datamap";
+export const SYSTEM_ROUTE = "/system";
+export const CONFIG_WIZARD_ROUTE = "/config-wizard";
```

Comment:
I think this block of constants is for API routes, `api/v1/{FOO_ROUTE}`, though I think I've seen them mixed around with the "UI ROUTES" below. @chriscalhoun1974 this is from the Ops codebase, right?

Reply:
@ssangervasi is correct in that this file is used within defining API routes in addition to root UI routes. It was a carryover from FidesOps.